templates/components/products/product-view.html

```diff
@@ -136,20 +136,21 @@ <h1 class="js-product-view-title c-product__title">
                 <input type="hidden" name="action" value="add">
                 <input type="hidden" name="product_id" value="{{product.id}}"/>
 
-                <div class="js-product-option-change" style="display:none;">
+                <div class="js-product-option-change c-product__options" style="display:none;">
                     {{inject 'showSwatchNames' theme_settings.show_product_swatch_names}}
                     {{#each product.options}}
                         {{{dynamicComponent 'components/products/options'}}}
                     {{/each}}
                 </div>
-                <div class="js-form-field js-form-field-stock{{#unless product.stock_level}} u-hidden-visually{{/unless}}">
+
+                {{!-- <div class="js-form-field js-form-field-stock{{#unless product.stock_level}} u-hidden-visually{{/unless}}">
```

Comment:
Remove commented code? :D

Reply:
okay, will remove since we don't show the stock in our current design